lib/pg_party.rb

```diff
@@ -37,9 +37,15 @@ def reset
   require "active_record/tasks/postgresql_database_tasks"
   require "pg_party/hacks/postgresql_database_tasks"
 
-  ActiveRecord::Tasks::PostgreSQLDatabaseTasks.prepend(
-    PgParty::Hacks::PostgreSQLDatabaseTasks
-  )
+  if VERSION::MAJOR >= 8 && VERSION::MINOR >= 1
```

Comment:
Where does this information come from? Maybe it would be better this way?

```suggestion
  if ActiveRecord::VERSION::MAJOR >= 8 && ActiveRecord::VERSION::MINOR >= 1
```

Or:


```suggestion
  if Gem::Version.new(ActiveRecord::VERSION::STRING) >= Gem::Version.new("8.1.0")
```